src/lazy/text/matched.rs

```diff
@@ -178,7 +204,7 @@ pub(crate) struct MatchedDecimal {
     is_negative: bool,
     digits_offset: u16,
     digits_length: u16,
-    trailing_digits_length: u16,
+    num_trailing_digits: u16,
```

Comment:
🗺️ Now that underscores have been filtered out of the trailing digits, this has been renamed to reflect that this is a digit count, not a length count.